app/src/components/Layers/TracksLayer.js

```diff
@@ -99,18 +104,20 @@ export default class TracksLayerGL {
     this.stage.lineStyle(0);
 
     // inner range center circle
-    this.stage.beginFill('0xFFFFFF', 1);
-    for (let i = 0, circlesLength = circlePoints.inner.x.length; i < circlesLength; i++) {
-      this.stage.drawCircle(circlePoints.inner.x[i], circlePoints.inner.y[i], 2);
-    }
-    this.stage.endFill();
+    if (drawParams.zoom > MIN_ZOOM_LEVEL) {
```

Comment:
If drawParams.zoom < MIN_ZOOM_LEVEL, you don't need to collect `circlePoints` in the loop above. Maybe that if should be a bit above (then just check for `circlePoints`'s existence.